Firestore/core/src/firebase/firestore/core/firestore_client.cc

```diff
@@ -377,11 +384,12 @@ void FirestoreClient::GetDocumentsFromLocalCache(
   auto shared_callback = absl::ShareUniquePtr(std::move(callback));
   auto shared_this = shared_from_this();
   worker_queue()->Enqueue([shared_this, query, shared_callback] {
-    DocumentMap docs = shared_this->local_store_->ExecuteQuery(query.query());
+    QueryResult query_result = shared_this->local_store_->ExecuteQuery(
```

Comment:
Port of https://github.com/firebase/firebase-android-sdk/blob/d8ff6a54d12fd676a31c2c331f09fe0a7d033202/firebase-firestore/src/main/java/com/google/firebase/firestore/core/FirestoreClient.java#L209